SuperBuild/External_python-dicom-requirements.cmake

```diff
@@ -40,41 +40,31 @@ endif()
 if(NOT Slicer_USE_SYSTEM_${proj})
   set(requirements_file ${CMAKE_BINARY_DIR}/${proj}-requirements.txt)
   file(WRITE ${requirements_file} [===[
-  # [pydicom]
-  pydicom==2.2.1 --hash=sha256:444b5b7289135ff5ea76dfc69d3597dcfde1cd050ca387f709d777f35701242d
-  # [/pydicom]
-  # [Pillow]
+  pydicom==2.2.2 --hash=sha256:6ecb9c6d56a20b2104099b8ef8fe0f3664d797b08a0e0548fe0311b515b32308
```

Comment:
Did you use the [python_package_updater.py](https://github.com/Slicer/Slicer/blob/c72d890515ccca70e4173dfcedd370e197c44ae9/Utilities/Scripts/python_package_updater.py) to update all the hashes here? That script relies on the entries in the generated requirements file to be within `# [pydicom]` and `# [/pydicom]` blocks and is used for maintenance purposes for updating hashes automatically.
https://github.com/Slicer/Slicer/blob/c72d890515ccca70e4173dfcedd370e197c44ae9/Utilities/Scripts/python_package_updater.py#L96-L97

Reply:
I had some trouble running the script and ended up pasting the results by hand. I will try it again.

Reply:
I've reverted the compatibility with python_package_updater.py
The problem was that I needed python 3.9 to run and the script and get the hashes, but I was building it so I needed to change the script a little bit and ended up messing the blocks. good catch.